AUTHORS

```diff
@@ -4,4 +4,31 @@
 # especially since many employees of one corporation may be contributing.
 # To see the full list of contributors, see the revision history in
 # source control.
-Google LLC
+
+# KFP v0.1.0 authors
+
+#### Frontend:
+Yasser Elsayed  https://github.com/yebrahim
+Riley Bauer     https://github.com/rileyjbauer
+
+#### Backend:
+Yang Pan        https://github.com/IronPan
+Pascal Vicaire  https://github.com/vicaire
+
+#### SDK:
+Bradley Jiang   https://github.com/qimingj
+Alexey Volkov   https://github.com/Ark-kun
+Ning Gao        https://github.com/gaoning777
```

Comment:
Jiaxiao Zheng https://github.com/numerology

Reply:
I haven't checked the history, but was Hongye Sun (https://github.com/hongye-sun) focusing on SDK as well?

Reply:
Okay, I misread. Just realize that this is a list for 0.1.0 only. In that case, are we going to have lists for 0.2, 0.3, etc. as well?

Reply:
Sorry for the confusion. I was in the process of preparing the whole list (with you, Yuan, Jiaxiao, Hongye, Renmin, Jing, Rui, etc plus external contributors), but have not finished it yet.

I was really sad when I saw that Jiaxiao is now missing from the GitHub KFP contributors due to removal of his @google.com e-mail =(

Reply:
Funny story, I noticed that Jiaxiao disappeared from the contributors list and messaged him. We retrospected and figured that unbinding his @google.com email was likely the cause. (To me, it feels like a GitHub bug).
After that, I prompted our new devs to not repeat the same mistake.

Reply:
This makes sense from GitHub's perspective - Git commits only have names and e-mails and they can be arbitrary.
GitHub tries to link commits to user account by e-mail, but cannot do so if the no GitHub users have this e-mail.

Maybe GitHub could have cached that information...

Reply:
I've now updated the file to include all contributors (with 5+ commits).